esda/crand.py

```diff
@@ -35,7 +35,7 @@ def intercepted_function(f, *f_args, **f_kwargs):
 
 
 @njit(fastmath=True)
-def vec_permutations(max_card: int, n: int, k_replications: int, seed=None):
```

Comment:
I think this needs to read 
`def vec_permutations(max_card: int, n: int, k_replications: int, seed: int)` as a typehint. This code will pass the integer *type itself* down to the function, which I don't think will work as a default argument to numpy.random.seed?

Reply:
That's what's changed? 

https://github.com/ljwolf/esda/pull/9/files#diff-1fd239570b4623d6ce23f0e268495633R38

Reply:
nm just saw the problem, one sec